scripts/validate-user-analytics.ts

```diff
@@ -26,6 +26,7 @@ const metricsTableRe = /<!--METRICS_TABLE_BEGIN-->([\s\S]*)<!--METRICS_TABLE_END
 function _exec(command: string, args: string[], opts: { cwd?: string }, logger: logging.Logger) {
   const { status, error, stdout } = spawnSync(command, args, {
     stdio: ['ignore', 'pipe', 'inherit'],
+    maxBuffer: 1024 * 1024,
```

Comment:
This change should be in it's own commit.

Reply:
Done

Reply:
Yesterday I was looking at a similar problem in https://github.com/angular/angular-cli/pull/15176/commits/e4e04e24ac006c53f9644a4007133c080e7fd675 and now I know that `1024 * 1024` is actually the default. 

Is this change actually doing something? I guess only if `maxBuffer` was being passed as a smaller value in `opts` but I don't find `maxBuffer` referenced anywhere in scripts.

Reply:
Looks like a leftover.

Reply:
Not a leftover. On node 10.x it's smaller so this change is required.